pyglmnet/datasets.py

```diff
@@ -43,7 +44,37 @@ def _reporthook(count, block_size, total_size):
                      % (percent, progress_size / (1024 * 1024)))
 
 
-def fetch_tikhonov_data(dpath):
+def _read_json(dpath, file_name):
+    """Function to read JSON file from a given ``dpath``"""
+    with open(op.join(dpath, file_name), 'r') as f:
+        dataset = json.loads(f.read())
+    return dataset
```

Comment:
I think a function for this is an overkill. I would remove this and read the file in the example